src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2792,6 +2916,15 @@ class PvtModelsDescription:
         where the original PVT file cannot be guaranteed to exist therefore the only reproducible way to recreate
         the PVT is trough the PvtModelTableParametersDescription.
 
+    :ivar ph_table_parameters:
+        *INTERNAL USE ONLY*
+
+        This attribute is populated when exporting a Study to a CaseDescription, and it holds a model representation
+        of a PVT originated from a (.tab / .alfatable) file.
```

Comment:
Don't we have a different pvt table extension? 

Even if we do not have. Perhaps add information about what kind of table this attribute holds. Something like:
```
This attribute is populated when exporting a Study to a CaseDescription, and it holds a model representation
of a PVT originating from a (.tab / .alfatable) file. Also, it holds values from a Pressure-Enthalpy PVT table,
in which temperature is one of the output/state variables.
```

Reply:
For now we have the same suffix `.tab` for `PT` or `PH` tables.

Nice, I will complement the information.